x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -139,6 +139,15 @@ export const links: LinkItem = {
           defaultMessage: 'Exception lists',
         }),
       ],
+      // links: [
```

Comment:
is this intentional?

Reply:
nope I'll remove it thank you!

Reply:
Fixed here: 3c63c44900387a6d7f468a59295bd3d572756fbd